packages/dds/sequence/src/sharedNumberSequence.ts

```diff
@@ -7,6 +7,11 @@ import { IFluidDataStoreRuntime, IChannelAttributes } from "@fluidframework/data
 import { SharedNumberSequenceFactory } from "./sequenceFactory";
 import { SharedSequence } from "./sharedSequence";
 
+/**
+ * The SharedNumberSequence holds a sequence of numbers. Each number will be stored
+ * at a position within the sequence. See the [sequence readme](../../README.md) for details on
```

Comment:
@tylerbutler will this link work? Is there a different format? Tried looking for docs, but didn't find anything that really converage authoring docs

Reply:
Not sure! Researching...

Reply:
Here's the correct link format: https://api-extractor.com/pages/tsdoc/tag_link/

But I don't know how relative links would be resolved. Best thing to do is use a absolute GitHub link for now, I think.